src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPage.epics.js

```diff
@@ -122,3 +124,36 @@ export const openEnrollmentPageEpic = (action$: InputObservable, store: ReduxSto
         },
         ),
     );
+
+export const clickLinkedRecordEpic = (action$: InputObservable, store: ReduxStore, { history }: ApiUtils) =>
+    action$.pipe(
+        ofType(enrollmentPageActionTypes.LINKED_RECORD_CLICK),
+        flatMap(({ payload }) => {
+            let url;
+            const { programId, orgUnitId } = payload;
+            if (payload.eventId) {
+                const recordProgram = getProgramFromProgramIdThrowIfNotFound(programId);
+                if (recordProgram instanceof EventProgram) {
+                    url = `/viewEvent?viewEventId=${payload.eventId}`;
+                } else {
+                    url = `/enrollmentEventEdit?${buildUrlQueryString({
+                        orgUnitId,
+                        programId,
+                        eventId: payload.eventId,
+                    })}`;
+                }
+            } else if (payload.teiId) {
+                url = `/enrollment?${buildUrlQueryString({
```

Comment:
Hey, 
What happens when the program where we redirect the user doesn't have the Enrollment dashboard feature enabled? Should the user be redirected to the old tracker in this scenario? I have the same doubt for `enrollmentEventEdit`, if the whole Enrollment dashboard feature is not enabled should the user be able to reach the enrollment event?
Thanks!

Reply:
hey, I don't think user can reach the relationship widget page ( on enrollment dashboard or enrollment event edit page) without having Enrollment dashboard feature enabled?

Reply:
Yes, the user cannot access the relationship widget page without having the Enrollment dashboard feature enabled. 

What I meant is when the user clicks to view a TEI relationship and this TEI has a different `programId` than the current context. The feature is enabled/disabled per program, so if we change the program, shouldn't we check what navigation is allowed in the new program context?
Thanks!

Reply:
@simonadomnisoru that's tricky because enrollment event edit only available in capture app. I will take this question to tracker-capture chanel to see if we want to implement it differently

Reply:
@simonadomnisoru discussed during the capture design meeting, we want to link it to the new capture for all the cases